src/components/DiagnosticsBody.vue

```diff
@@ -55,13 +63,38 @@ export default Vue.extend({
     timeLastRunTooltip(): string {
       return this.timeLastRun.toLocaleString();
     },
+    filteredRows(): any {
+      if (!this.hideMuted) {
+        return this.rows;
+      }
+
+      return this.rows.filter(x => x.mute === false);
+    },
+    hasMutedResults(): boolean {
+      return !!this.rows.length && this.hideMuted;
```

Comment:
This will be true if there are only unmuted results; that doesn't match the function name.
Did you want to actually look for one?
```suggestion
      return this.hideMuted && this.rows.some(x => x.mute);
```
Depending on what we want, we may drop the first half of that condition…

Reply:
I guess this works with the `no-rows` slot so we could identify that we have results that are muted when there are no data to render vs. just having no rows returned. 

I thought that `hasMutedResults` kind of covered that pretty well. Maybe it makes sense to make a `no-content` component for diagnostics and that will work to clarify the context?

Reply:
I opted to rename this function to `areAllRowsMuted` and updated the logic to check for every row having `mute`.